### DISCUSSION TITLE
Monetary Discrepancies (GENERAL) [C]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#85] [Card Balance]
![image](https://user-images.githubusercontent.com/83040170/149008883-495e929e-74bf-4859-baae-56da8b4abd7a.png)

DoomTrooper 2nd edition

MONETARY DISCREPANCIES
General Special
ATTACH TO ANY PLAYER AT ANY TIME. While attached, each time the affected player wishes to play any Equipment, Fortification, Relic, Warzone, or Special card, he or she must spend 2D (in addition to any costs from the card itself). Multiple attachments of this card are NOT cumulative. (+)

My suggestion based on this version :

Monetary Discrepancies (GENERAL) [C]
INFLUENCE, UNIQUE
Your opponent must spend 2 [DP] (in addition to any costs from the card itself) to play fate cards during 
POSTCOMBAT phase. 

🤡

**Comment by TomekPL-1** [comment]
Monetary Discrepancies (Bauhaus) [C]
INFLUENCE
Play for 1 Action and 3 [DP]. While in play, every player must pay 1[DP] any time he wishes to play any FATE card. Multiply Monetary Discrepancies are cumulative.

**Reply by Thorgalski** [reply]
I agree make it Bauhaus.

**Reply by 21-ric** [reply]
I really love this version. A nice card for Bauhaus. 👍
